src/go/rpk/pkg/cli/cmd/container/common/common.go

```diff
@@ -256,8 +256,15 @@ func CreateNode(
 		HostAddr(proxyPort),
 		"--advertise-rpc-addr",
 		fmt.Sprintf("%s:%d", ip, config.Default().Redpanda.RPCServer.Port),
+		"--enable-wasm",
 		"--smp 1 --memory 1G --reserve-memory 0M",
 	}
+	entrypoint := []string{"/entrypoint.sh"}
+	if enableWasm {
+		cmd = append(cmd, "--enable-wasm", len(cmd) - 1);
+		entrypoint = []string{"/entrypoint-wasm.sh"}
```

Comment:
Why does it need a new entrypoint?

Reply:
This allows this logic to instead be executed at startup:
https://github.com/vectorizedio/vtools/pull/176/files#diff-65b6afb830965b907f8124de74adef503b0038399faecab626d6e304d5e5dc54